sdcm/nemesis.py

```diff
@@ -424,7 +422,18 @@ def _get_target_nodes(
         """
         if is_seed is DefaultValue:
             is_seed = False if self.filter_seed else None
-        nodes = [node for node in self._target_node_pool if not node.running_nemesis]
+
+        match self._target_node_pool:
```

Comment:
target_node_pool will  always be data_type (in most cases), because set target nodes is executed in disrupt_wrapper, which is first in decorator changes, and decorator which will change the node pool type, will change it after target node was chosen